### ISSUE TITLE
[Q] How to plot val losses during training?

### ISSUE THREAD

**Issue by David-Biggs** [wandb/wandb#3728]
Thanks for an awesome tool!! I have a question regarding plotting validation losses:

val/loss is not an option in the drop-down menu for adding a pane:

![image](https://user-images.githubusercontent.com/64862726/171134791-6c07f60c-1679-45b4-b5b2-9677f004280e.png)

But the val losses are logged.  A section of the output.log file is shown below where the val losses are highlighted: 

![image](https://user-images.githubusercontent.com/64862726/171133981-2edf52b5-8856-4c2a-8384-1a1d442ca65a.png)

How do I go about plotting the val losses?

Many thanks

**Comment by David-Biggs** [comment]
Hi @nate-wandb, thanks for following up.

When I wrote the issue I could not find val/loss in the dropdown. However, a few days later, it appeared. 

Thanks again!
